package.json

```diff
@@ -1,29 +1,58 @@
 {
-  "name": "mobile-examiner-alpha-journal-service",
-  "version": "1.0.0",
-  "description": "DVSA Mobile Examiner Services (GDS Alpha phase) - Journal Microservice",
-  "dependencies": {
-    "redis": "^2.8.0"
-  },
-  "devDependencies": {
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "copyfiles": "^1.2.0"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "copy-package": "./node_modules/.bin/copyfiles package.json build",
-    "compile": "rm -rf build && ./node_modules/.bin/babel src -d build"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/dvsa/mobile-examiner-alpha-journal-service.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/dvsa/mobile-examiner-alpha-journal-service/issues"
-  },
-  "homepage": "https://github.com/dvsa/mobile-examiner-alpha-journal-service#readme"
-}
+    "name": "mobile-examiner-alpha-journal-service",
+    "version": "1.0.0",
+    "description": "DVSA Mobile Examiner Services (GDS Alpha phase) - Journal Microservice",
+    "dependencies": {
+        "@types/aws-lambda": "0.0.5",
+        "@types/aws-sdk": "0.0.40",
+        "@types/chai": "^4.1.2",
+        "@types/mocha": "^2.2.48",
+        "@types/redis": "^2.8.6",
+        "redis": "^2.8.0"
+    },
+    "devDependencies": {
+        "@types/jest": "^22.2.0",
+        "babel-cli": "^6.26.0",
```

Comment:
do we still need babel? I think TS will now be responsible for transpiling to ESX